### PR TITLE
Added current-status doc for op-codes wg

### PR DESCRIPTION
**Pull request by jasonbcox**

### CODE REVIEW

File: wg-opcodes/summaries/current-status.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## OpCodes Under Discussion or Previously Discussed
+
+OP_DATASIGVERIFY
+ * Spec and implementation proposed, but not agreed upon.  See: https://github.com/shadders/uahf-spec/pull/6/files
+OP_GROUP
+ * Original proposal: https://medium.com/@g.andrew.stone/bitcoin-scripting-applications-representative-tokens-ece42de81285
+ * BUIP 077: https://github.com/BitcoinUnlimited/BUIP/blob/cd0763223fb79b0388b7d307bf4b2acdb9fc9a56/077.mediawiki
+ * Requirements proposed: https://www.yours.org/content/on-representative-tokens--colored-coins--bb7a829b965c/
+ * Implementation proposed to continue discussion: https://github.com/gandrewstone/BitcoinUnlimited/commit/7b8d8b708d1b15d0e05d6ef7e6f0dc1799efe186#diff-799cde8829295bd34ad577deb16cc152
```
Comment (jasonbcox) [inline wg-opcodes/summaries/current-status.md]: This is the best link I could find.  Let me know if there is a better one.

File: wg-opcodes/summaries/current-status.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Workgroup:        OpCodes
+Document Version: 1.0
+
+# Document Purpose
+
+This document is intended to provide a single source of truth of the status of ongoing op codes
+discussions relative to the upcoming hardfork. Any discussions not related to the latest upcoming
+fork will be referenced and postponed until the next hardfork or removed entirely if the discussion
+is in a complete form and no spec is planned to be proposed.  Implementation details or specs of
+particular op codes is out of scope of this document.
```
Comment (jasonbcox) [inline wg-opcodes/summaries/current-status.md]: I may reword this last sentence to something like:
Implementation details or specs of particular op codes is out of scope of this document, however they will be referenced.

File: wg-opcodes/summaries/current-status.md (hunk 1 above)
Comment (sickpig) [reply wg-opcodes/summaries/current-status.md]: branch w/ consensus code change only:
https://github.com/gandrewstone/BitcoinUnlimited/commits/opgroup_consensus

branch containing more wide code changes including e.g. wallet update:
https://github.com/gandrewstone/BitcoinUnlimited/commits/opgroup_fullnode

File: wg-opcodes/summaries/current-status.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+OP_NUM2BIN
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_OR
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_SPLIT
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_XOR
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+
+## OpCodes Under Discussion or Previously Discussed
```
Comment (jasonbcox) [inline wg-opcodes/summaries/current-status.md:40]: Need to add OP_MUL and OP_INVERT.  Still need to dig up discussions on these, but it looks like OP_MUL is tentative for the Nov 2018 fork and OP_INVERT may be unnecessary.

**Comment by schancel** [comment]
Can this be merged now, or are the updates that need to be made?

**Review by Danconnolly — APPROVED** [review]

File: wg-opcodes/summaries/current-status.md (hunk 1 above)
Comment (schancel) [reply wg-opcodes/summaries/current-status.md]: Do these links need to go in before we merge?

File: wg-opcodes/summaries/current-status.md (hunk 3 above)
Comment (shadders) [reply wg-opcodes/summaries/current-status.md:40]: Not sure why op_invert is unnecessary?  It was put off from May to limit scope to a manageable size but not heard anyone say it's unnecessary.

File: wg-opcodes/summaries/current-status.md (hunk 1 above)
Comment (sickpig) [reply wg-opcodes/summaries/current-status.md]: Don't have a strong opinion, but the provided links seems to give a more comprehensive view of the changes needed, both at the consensus and wallet level.

File: wg-opcodes/summaries/current-status.md (hunk 3 above)
Comment (jasonbcox) [reply wg-opcodes/summaries/current-status.md:40]: Someone had suggested that OP_INVERT may not be necessary with the inclusion of OP_REPEAT or OP_BIN2NUM.  I haven't really verified these claims though.  I'll go ahead and include OP_INVERT for later investigation and not mention the unnecessary bit.

**Pull request merged**